Unit3/README.md

```diff
@@ -11,7 +11,8 @@
 ### Mini-Projects
 
 * Included in OC course:
-   * Mini-Project 3.1: [Add a Validation Message](https://openclassrooms.com/en/courses/4286486-build-web-apps-with-reactjs/4317901-practice-add-a-validation-message) and [Edit Existing Products](https://openclassrooms.com/en/courses/4286486-build-web-apps-with-reactjs/4317921-practice-extend-the-form-functionality-to-edit-existing-products)
+   * Mini-Project 3.1a: [Add a Validation Message](https://openclassrooms.com/en/courses/4286486-build-web-apps-with-reactjs/4317901-practice-add-a-validation-message)
```

Comment:
Similar comment as in my Unit 1 PR, make sure the links point to the READMEs in a relative way.